### DISCUSSION TITLE
Center of wake outside the low resolution grid in the x direction

### DISCUSSION THREAD

**Discussion by MYMahfouz** [OpenFAST/openfast#1699] [Q&A] [answered]
Hi,

I am receiving this warning in my log file. I understand what it means and I know that I set more wake planes than I extended the low resolution grid in the x direction. I want to be sure I am capturing the wake effect on the power production within the farm and I wouldn't analyze the wake dynamics or the downwind flow field.

I focus on the turbines power production, mooring lines loads and the floater motions in the 6 DoFs.

My question is if this warning affects my results or make my simulations slower? 

> FARM_CalcOutput:AWAE_CalcOutput:LowResGridCalcOutput:The center of wake plane #73 for turbine #8
has passed the upper-most X boundary of the low-resolution domain.
LowResGridCalcOutput:The center of wake plane #74 for turbine #8 has passed the upper-most X
boundary of the low-resolution domain.

Thank you in advance for your help.

Best regards,

**Comment by jjonkman** [comment, marked as answer]
Dear @MYMahfouz,

This warning indicates that wakes are exiting the downwind end of the low-resolution domain (assuming you are simulating with a wind direction of zero, along positive global X).  Overall, there should be no issue in FAST.Farm with wakes leaving the downwind end of the domain as that wont effect other simulation results, but it may indicate that you could reduce the number of wake planes to reduce computational expense.

Best regards,

**Reply by MYMahfouz** [reply]
Dear @jjonkman,

Thank you so much this answers my question. I decreased the number of wake planes as you suggested.

Best regards,
